src/core_plugins/vega/public/data_model/es_query_parser.js

```diff
@@ -168,7 +200,10 @@ export class EsQueryParser {
             if (size === true) {
               size = 50; // by default, try to get ~80 values
             } else if (typeof size !== 'number') {
-              throw new Error(`"${AUTOINTERVAL}" must be either true or a number`);
+              throw new Error(i18n.translate('vega.esQueryParser.autointervalValueTypeErrorMessage', {
+                defaultMessage: '"{autointerval}" must be either true or a number',
```

Comment:
Property values (here `true`) should also be pulled out to params

Reply:
agree, `true` should be a param